tcms/static/js/utils.js

```diff
@@ -24,6 +24,12 @@ function updateSelect(data, selector, id_attr, value_attr) {
     }
 }
 
+function configure_product_on_load() {
+    $('#id_product').change(function() {
+        $('#id_category').find('option').remove();
```

Comment:
For some reason this didn't work for me. When adding a new Product the category list contained --default-- 2 times, the 1st entry being from the previous product.

Reply:
Also happens when changing products without adding new ones. Maybe the copy of this function is not exactly what you need.